app/views/dashboard/_pending_applications.haml

```diff
@@ -10,5 +10,6 @@
               %li= link_to record.user.proper_name, record
           - else
             No pending applications
-        = link_to 'View saved applications for ' + position.name,
+        - count = position.saved_applications.count
```

Comment:
Here's the thing: The `position` referenced here is something being iterated through in the view. There is currently no way to turn `position.saved_applications` into an instance variable, given that we get it from an instance variable called `@departments`.